lib/chef/secret_fetcher/azure_key_vault.rb

```diff
@@ -48,6 +49,8 @@ def do_fetch(name, version)
         end
       end
 
+      private
```

Comment:
No idea why this wasn't marked as private before. Assuming this is OK, but let me know if there was a reason not to mark all of these methods as private.